lib/fishjam_web/controllers/component_controller.ex

```diff
@@ -129,8 +132,11 @@ defmodule FishjamWeb.ComponentController do
          :ok <- Room.remove_component(room_id, id) do
       send_resp(conn, :no_content, "")
     else
-      {:error, :room_not_found} -> {:error, :not_found, "Room #{room_id} does not exist"}
-      {:error, :component_not_found} -> {:error, :not_found, "Component #{id} does not exist"}
+      {:error, :component_not_found} ->
+        {:error, :not_found, "Component #{id} does not exist"}
+
+      other ->
```

Comment:
If I remember correctly there is no need to have this `other -> other` fallback should automatically catch those.

Reply:
Not in this case :c
```

  1) test delete peer deletes peer from not exisiting room (FishjamWeb.PeerControllerTest)
     test/fishjam_web/controllers/peer_controller_test.exs:132
     ** (WithClauseError) no with clause matching: {:error, :room_not_found}
     code: conn = delete(conn, ~p"/room/#{room_id}/peer/#{peer_id}")
     stacktrace:
       (fishjam 0.6.3) lib/fishjam_web/controllers/peer_controller.ex:120: FishjamWeb.PeerController.delete/2
       (fishjam 0.6.3) lib/fishjam_web/controllers/peer_controller.ex:1: FishjamWeb.PeerController.action/2
       (fishjam 0.6.3) lib/fishjam_web/controllers/peer_controller.ex:1: FishjamWeb.PeerController.phoenix_controller_pipeline/2
       (phoenix 1.7.12) lib/phoenix/router.ex:484: Phoenix.Router.__call__/5
       (fishjam 0.6.3) lib/fishjam_web/endpoint.ex:1: FishjamWeb.Endpoint.plug_builder_call/2
       (fishjam 0.6.3) lib/fishjam_web/endpoint.ex:1: FishjamWeb.Endpoint.call/2
       (phoenix 1.7.12) lib/phoenix/test/conn_test.ex:225: Phoenix.ConnTest.dispatch/5
       test/fishjam_web/controllers/peer_controller_test.exs:134: (test)

     The following output was logged:
     12:53:12.978 room_id=1e26932f-8f32-4cf2-9159-f48e120b82ef [info] Initialize room
     12:53:12.978 [info] Created room "1e26932f-8f32-4cf2-9159-f48e120b82ef"
     12:53:12.978 room_id=1e26932f-8f32-4cf2-9159-f48e120b82ef [info] Added peer "9689ddf6-fbb1-43c7-afa5-8636023fceac"
     12:53:12.997 room_id=1e26932f-8f32-4cf2-9159-f48e120b82ef [info] Removed peer "9689ddf6-fbb1-43c7-afa5-8636023fceac" from room "1e26932f-8f32-4cf2-9159-f48e120b82ef"
     12:53:12.997 [info] Deleted room "1e26932f-8f32-4cf2-9159-f48e120b82ef"


```

Reply:
Ah it works if there is no `else` clause completely then results goes to fallback, my bad :)